test/Makefile

```diff
@@ -43,10 +43,10 @@ include ../common.mk
 
 # Testing mode option (quick/thorough)
 ifeq ($(quickertest), 1)
-	NVCCFLAGS += -DQUICKER_TEST
+	NVCCFLAGS += -DMINIMAL_TEST
```

Comment:
This doesn't match the `#ifdef`s, should be `CUB_TEST_MINIMAL`.

Reply:
Can we just delete the makefiles at this point? Who uses them? We don't.

Reply:
Burn them 🔥🔥🔥